### ISSUE TITLE
crc setup failing at INFO Caching crc-admin-helper executable

### ISSUE THREAD

**Issue by adithyagk** [code-ready/crc#2870]
I was trying to install Code ready containers on Centos which is running on VMware, The issue occurs when i try to trigger **crc setup** command.
It fails to find a release called 0.0.6? 
May i know how to mitigate this?

### General information

  * OS: Linux
  * Hypervisor: KVM
  * Did you run `crc setup` before starting it (Yes/No)? Yes
  * Running CRC on: VM (Windows running VMware with Centos)

## CRC version
```bash
# Put `crc version` output here
`[adithya@localhost ~]$ crc version
WARN A new version (1.36.0) has been published on https://developers.redhat.com/content-gateway/file/pub/openshift-v4/clients/crc/1.36.0/crc-linux-amd64.tar.xz 
CodeReady Containers version: 1.32.1+4d40571f
OpenShift version: 4.8.5 (not embedded in executable)
`
```
  
## CRC status
```bash
# Put `crc status --log-level debug` output here
`[adithya@localhost ~]$ crc status --log-level debug
DEBU CodeReady Containers version: 1.32.1+4d40571f 
DEBU OpenShift version: 4.8.5 (not embedded in executable) 
DEBU Running 'crc status'                         
DEBU Checking file: /home/adithya/.crc/machines/crc/.crc-exist 
Machine does not exist. Use 'crc start' to create it
`
```

## CRC config
```bash
# Put `crc config view` output here
`[adithya@localhost ~]$ crc config view
- consent-telemetry                     : no
`
```

## Host Operating System
```bash
# Put the output of `cat /etc/os-release` in case of Linux
`[adithya@localhost ~]$ cat /etc/os-release
NAME="CentOS Linux"
VERSION="7 (Core)"
ID="centos"
ID_LIKE="rhel fedora"
VERSION_ID="7"
PRETTY_NAME="CentOS Linux 7 (Core)"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:centos:centos:7"
HOME_URL="https://www.centos.org/"
BUG_REPORT_URL="https://bugs.centos.org/"

CENTOS_MANTISBT_PROJECT="CentOS-7"
CENTOS_MANTISBT_PROJECT_VERSION="7"
REDHAT_SUPPORT_PRODUCT="centos"
REDHAT_SUPPORT_PRODUCT_VERSION="7"
`
```

### Steps to reproduce

  1. run crc setup

### Expected

Crc setup should proceed as expected

### Actual
`[adithya@localhost ~]$ crc setup
INFO Checking if running as non-root              
INFO Checking if running inside WSL2              
INFO Checking if crc-admin-helper executable is cached 
INFO Caching crc-admin-helper executable          
Unable to download crc-admin-helper executable: download of https://github.com/code-ready/admin-helper/releases/download/0.0.6/crc-admin-helper-linux failed: server returned 404 Not Found`

### Logs

Before gather the logs try following if that fix your issue
```bash
$ crc delete -f
$ crc cleanup
$ crc setup
$ crc start --log-level debug
```

Please consider posting the output of `crc start --log-level debug`  on http://gist.github.com/ and post the link in the issue.
Issue persists before crc start
```


```

**Comment by gbraad** [comment]
please provide more information. the issue template has not been followed

**Comment by adithyagk** [comment]
May i please know what is the standard template since iam new here?

**Comment by gbraad** [comment]
> Unable to download crc-admin-helper executable: download of https://github.com/code-ready/admin-helper/releases/download/0.0.6/crc-admin-helper-linux failed: server returned 404 Not Found

Also note that CRC is supposed to be downloaded from the Red Hat Developer website; https://developer.redhat.com/ It seems the needed binary is not embedded, so likely you are not using the right download (or have build one yourself)

**Comment by gbraad** [comment]
> template

https://github.com/code-ready/crc/issues/new?assignees=&labels=kind%2Fbug%2C+status%2Fneed+triage&template=bug_report.md&title=%5BBUG%5D

the information you Ctrl+A removed after clicking the "New issue" button ?

**Comment by adithyagk** [comment]
Hi @gbraad, i downloaded the standard tar file using wget https://mirror.openshift.com/pub/openshift-v4/clients/crc/latest/crc-linux-amd64.tar.xz
i extracted it, placed the crc on bin folder.
Ran crc setup right after

**Comment by gbraad** [comment]
The template asked basic questions we need to help you. Such as, on which platform are you? How much memory. The version of the binary, etc.

Please provide as much detail as possible. I'll verify in the mean time the release you downloaded.

**Comment by adithyagk** [comment]
Hi @gbraad  I have updated it with the template.

**Comment by adithyagk** [comment]
Hi @gbraad  it seems to have been an issue associated with a broken binary, i have retired it and it seems to be ok
Thanks
